### DISCUSSION TITLE
Installing Red Hat OpenShift Pipelines

### DISCUSSION THREAD

**Discussion by planetf1** [crc-org/crc#4365] [General]
I was interested in looking at OpenShift Pipelines. - for example following instructions similar to https://docs.openshift.com/container-platform/4.9/cicd/pipelines/installing-pipelines.html

I have CRC 2.41.0+e6495f setup, in openshift mode, but when I go to the OperatorHub, I see no sign of pipelines.

Is this viable - just for education/experimentation?

**Comment by praveenkumar** [comment]
@planetf1 what is the host OS you are using (mac/windows/linux)? Also https://docs.openshift.com/pipelines/1.15/install_config/installing-pipelines.html is correct resource since pipeline release is async with OpenShift.

**Reply by planetf1** [reply]
macOS.

I recreated the crc environment (4.16.7) and can see openshift pipelines now. So not sure on cause, but can now close. Thanks
